rollup.config.js

```diff
@@ -19,11 +19,13 @@ export default {
       sourcemap: true
     }
   ],
-  external: ['crypto'],
   plugins: [
     peerDepsExternal(),
-    resolve(),
+    resolve({
+        browser: true, // This instructs the plugin to use
+                       // the "browser" property in package.json
```

Comment:
Where is the `browser` property?

Reply:
https://github.com/uuidjs/uuid/blob/master/package.json#L35 I think this is the one that causes issues for us (rng module)

Reply:
How about specifying a dependency?
- The "browser" field in a dependency's package.json